src/main/python/mmlspark/lightgbm/LightGBMClassifier.py

```diff
@@ -34,33 +34,45 @@ def saveNativeModel(self, filename, overwrite=True):
     @staticmethod
     def loadNativeModelFromFile(filename, labelColName="label",
                                 featuresColName="features", predictionColName="prediction",

```

Comment:
can we remove the col clutter on these? is there some way to set the cols in a unified way afterward?

Reply:
done